### PR TITLE
Groom branch for 1.2.2-beta.1

### PR DESCRIPTION
**Pull request by prprabhu-ms**
TODO:

* Groom the CHANGELOG file.
* Delete the GROOMME marker file.

Merge this PR *before* kicking off the release branch creation.

### CODE REVIEW

File: packages/communication-react/CHANGELOG.md [hunk 1]
```diff
@@ -7,114 +7,50 @@ This log was last generated on Tue, 19 Apr 2022 20:46:13 GMT and should not be m
...
 
-### Minor changes
+### Features
 
-- `@internal/react-composites`
-  - Add useAzureCommunicationChatAdapter hook ([PR #1627](https://github.com/azure/communication-ui-library/pull/1627) by 82062616+prprabhu-ms@users.noreply.github.com)
-- `@internal/react-components`
-  - Introduce strings for the new modal controls. ([PR #1718](https://github.com/azure/communication-ui-library/pull/1718) by 94866715+dmceachernmsft@users.noreply.github.com)
+  - Show Error Bar to user when joining a call fails ([PR #1788](https://github.com/azure/communication-ui-library/pull/1788) by 2684369+JamesBurnside@users.noreply.github.com)
+  - Added local and remote picture-in-picture component in Chat pane of CallWithChat composite in mobile view. ([PR #1617](https://github.com/azure/communication-ui-library/pull/1617) by miguelgamis@microsoft.com)
```
Comment (prprabhu-ms) [inline packages/communication-react/CHANGELOG.md:15]: Can we make the verb tense consistent across these. Pick one of "Added" vs "Add" etc.

**Comment by prprabhu-ms** [comment]
Can't approve my own PR. Assigned a couple folks.

**Comment by prprabhu-ms** [comment]
btw, this PR can not pass CI right now. There are now bugs open to fix this in the prerelease branch creation workflow (and a PR up for that). For this one, just merge it without CI being green when you're ready.

File: packages/communication-react/CHANGELOG.md [hunk 2]
```diff
@@ -7,114 +7,50 @@ This log was last generated on Tue, 19 Apr 2022 20:46:13 GMT and should not be m
...
 
-### Minor changes
+### Features
 
-- `@internal/react-composites`
-  - Add useAzureCommunicationChatAdapter hook ([PR #1627](https://github.com/azure/communication-ui-library/pull/1627) by 82062616+prprabhu-ms@users.noreply.github.com)
-- `@internal/react-components`
-  - Introduce strings for the new modal controls. ([PR #1718](https://github.com/azure/communication-ui-library/pull/1718) by 94866715+dmceachernmsft@users.noreply.github.com)
+  - Show Error Bar to user when joining a call fails ([PR #1788](https://github.com/azure/communication-ui-library/pull/1788) by 2684369+JamesBurnside@users.noreply.github.com)
+  - Add local and remote picture-in-picture component in Chat pane of CallWithChat composite in mobile view. ([PR #1617](https://github.com/azure/communication-ui-library/pull/1617) by miguelgamis@microsoft.com)
```
Comment (JamesBurnside) [inline packages/communication-react/CHANGELOG.md:15]: This was in last stable release, can remove

**Review by JamesBurnside — APPROVED** [review]

File: packages/communication-react/CHANGELOG.md (hunk 2 above)
Comment (anjulgarg) [reply packages/communication-react/CHANGELOG.md:15]: PiP component?

Comment (anjulgarg) [reply packages/communication-react/CHANGELOG.md:15]: Changelogs don't have any mention of it.
Might be worth keeping

Comment (JamesBurnside) [reply packages/communication-react/CHANGELOG.md:15]: Yeah the changelog just has "Added CallWithChatComposite" -- as this had a lot of features that we didn't exhaustively this

**Review by anjulgarg — APPROVED** [review]

**Pull request merged**